src/components/Message/styled.ts

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
```

Comment:
we should support themes on all these guys too

Reply:
Done